Lib/fontTools/cffLib/specializer.py

```diff
@@ -1,6 +1,17 @@
 # -*- coding: utf-8 -*-
 
-"""T2CharString operator specializer and generalizer."""
+"""T2CharString operator specializer and generalizer.
+
+PostScript glyph drawing operations can be expressed in multiple different
```

Comment:
How about something like, "The Type2 Charstring Format defines several operators which combine multiple drawing commands into one, typically with a smaller representation."

Also it might be nice to have a brief explanation of why/how specialization and generalization are used.

Reply:
I'm not sure this is what this module does - I might be wrong about this, but as I understand it, it doesn't really combine multiple drawing commands into one, but substitutes e.g. four-argument `rlineto` with three-argument `hlineto`.

Reply:
Ah, right, I remembered multiple operator merging being discussed, but that was never implemented in specializer. Carry on :-)